test/e2e/e2e.go

```diff
@@ -175,6 +191,64 @@ var _ = Describe("validation", func() {
 			})
 		})
 	})
+
+	Context("Creating AddressPool", func() {
+		It("Should create addresspool CR successfully", func() {
+			By("By creating AddressPool CR")
+			addresspool := &metallbv1alpha.AddressPool{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "metallb.io/v1alpha1",
+					Kind:       "AddressPool",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      MetallbAddressPoolName,
+					Namespace: MetallbNameSpace,
+				},
+				Spec: metallbv1alpha.AddressPoolSpec{
+					Name:     "test1",
+					Protocol: "layer2",
+					Addresses: []string{
+						"1.1.1.1",
+						"1.1.1.100",
+					},
+				},
+			}
+
+			Expect(testclient.Client.Create(context.Background(), addresspool)).Should(Succeed())
+
+			key := types.NamespacedName{
+				Name:      MetallbAddressPoolName,
+				Namespace: MetallbNameSpace,
+			}
+
+			// Create addresspool resource
+			By("By checking AddressPool resource is created")
+			Eventually(func() error {
+				err := testclient.Client.Get(context.Background(), key, addresspool)
+				return err
+			}, timeout, interval).Should(Succeed())
+
+			// Checking ConfigMap is created
+			By("By checking ConfigMap is created")
+			Eventually(func() error {
+				_, err := testclient.Client.ConfigMaps(MetallbNameSpace).Get(context.Background(), MetallbConfigMapName, metav1.GetOptions{})
```

Comment:
I'd check the content of the configmap here, to verify that it is what we expect to be.

Reply:
I will verify configmap content and update teh diffs

Reply:
Can you check multiple variants using tables as I suggested in https://github.com/metallb/metallb-operator/pull/27#discussion_r656005962 ?